### PR TITLE
Feature/normalize css

### PR DESCRIPTION
**Pull request by GrumpyPirate**
Switches over to using third-party [normalize.css](https://necolas.github.io/normalize.css/) as the default set of cross-browser 'reset' styles, deprecating the house styles' `reset.less`.

Normalize-css is pulled into the project via Bower, so the [main-bower-files](https://www.npmjs.com/package/main-bower-files) functionality has first been merged in from [feature/main-bower-files](https://github.com/tangentsnowball/house-styles/tree/feature/main-bower-files) in order to automate the process of concatenation with the application `styles.css` and `styles-responsive.css`.

### CODE REVIEW

File: _styleguide.html [hunk 1]
```diff
@@ -1108,8 +1109,8 @@
 <footer class="footer">
 </footer>
 
-    <script src="static/dist/bower_components/jquery/dist/jquery.min.js"></script>
-    <!--[if lt IE 9 ]> 
+    <script src="static/dist/js/vendor.min.js"></script>
```
Comment (MrFirthy) [inline _styleguide.html]: Does minifying and concatenating all third party js files into one `vendor.js` file make the assumption that all third party dependencies will be required on every page? as this is often not the case.

Comment (ecobbold15) [inline _styleguide.html]: Only if left unconfigured - the idea is to bundle all of the plugins that you *know* will be required on every page, such as jQuery, Angular, etc. Everything else you could handle separately.

**Comment by MrFirthy** [comment]
Cool makes sense to me. Only outstanding issue is the same conflicts that the general clean up branch had. If you could solve those by merging master into this branch then I'm happy to merge.

I will also close the `feature/main-bower-files` PR once this is merged because it seems to contain everything that has in here plus a bit more.

**Comment by ecobbold15** [comment]
Good to merge :+1:

**Pull request merged**